src/parser/dep.fs

```diff
@@ -330,8 +330,9 @@ let collect_one (verify_flags: list<(string * ref<bool>)>) (verify_mode: verify_
     | Pragma _ ->
         ()
     | TopLevelModule lid ->
-        raise (Err (Util.format1 "Automatic dependency analysis demands one \
-          module per file (module %s not supported)" (string_of_lid lid true)))
+        ()
+//        raise (Err (Util.format1 "Automatic dependency analysis demands one \
+//          module per file (module %s not supported)" (string_of_lid lid true)))
```

Comment:
Since multiple modules per file really are a hard error when using the automatic dependency analysis (the default), this should probably be added in parser/dep.fs ; `collect_one` could just allocate a reference and error out if it his `TopLevelModule` more than once.